webapp/loggedin/modules/data/dataVis/dataQuery/components/tableContent/tableColumn.js

```diff
@@ -32,6 +33,15 @@ const TableColumn = (props) => {
     return `surveyForm.${nodeDefTypePrefix}.${colName}`
   }
 
+  const getColValue = (col) => {
+    const value = Object.prototype.hasOwnProperty.call(row, col) ? row[col] : null
+    if (value) {
```

Comment:
safer to check null in NuberUtils.formatDecimal

Reply:
ok